drivers/usb/device/usb_dc_stm32.c

```diff
@@ -69,6 +69,14 @@ static const struct pinctrl_dev_config *usb_pcfg =
 #define USB_OTG_HS_EMB_PHY (DT_HAS_COMPAT_STATUS_OKAY(st_stm32_usbphyc) && \
 			    DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs))
 
+#define USB_OTG_HS_ULPI_PHY (DT_HAS_COMPAT_STATUS_OKAY(usb_ulpi_phy) && \
+			    DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs))
+
+#if USB_OTG_HS_ULPI_PHY
+static const struct gpio_dt_spec ulpi_reset =
+	GPIO_DT_SPEC_GET_OR(DT_PHANDLE(DT_INST(0, st_stm32_otghs), phys), reset_gpios, { 0 });
```

Comment:
```c
        GPIO_DT_SPEC_GET_OR(DT_PHANDLE(DT_INST(0, st_stm32_otghs), phys), \                                                                  
                            reset_gpios, { 0 });
```

Reply:
When that is done checkpatch gives an error
```
LINE_CONTINUATIONS: Avoid unnecessary line continuations
File:drivers/usb/device/usb_dc_stm32.c
Line:77
```